src/Controls/src/Core/ContentPage/ContentPage.Android.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Microsoft.Maui.Controls
+{
+	public partial class ContentPage
+	{
+		static void MapInputTransparent(IPageHandler handler, ContentPage page)
+		{
+			if (handler.PlatformView is ContentViewGroup layout)
+			{
+				layout.InputTransparent = page.InputTransparent;
```

Comment:
What happens if you swap the contents? Or if you set input transparent and then set the content.